lib/types.d.ts

```diff
@@ -399,6 +399,8 @@ declare namespace XlsxPopulate {
     text: string;
     width: string;
     height: string;
+    textAlign: string;
+    horizontalAlignment: string
```

Comment:
To help the user of the library: do you think it makes sense to be more restrictive with the values we accept here, or that would be limiting?

https://learn.microsoft.com/en-us/dotnet/api/documentformat.openxml.vml.spreadsheet.horizontaltextalignment?view=openxml-3.0.1#remarks

https://developer.mozilla.org/en-US/docs/Web/CSS/text-align#formal_syntax

Reply:
yes you're absolutely right!